pkg/storageos/update_status.go

```diff
@@ -97,6 +142,18 @@ func isHealthy(health *storageosv1.NodeHealth) bool {
 	return false
 }
 
+func isListening(host string, port string, timeout time.Duration) bool {
+	conn, err := net.DialTimeout("tcp", net.JoinHostPort(host, port), timeout)
+	if err != nil {
+		fmt.Println("Connecting error:", err)
```

Comment:
We should use the logger if this needs to be logged.

Reply:
It doesn't need to be logged - removing